packages/world/mud.config.mts

```diff
@@ -99,8 +99,17 @@ export default mudConfig({
       },
       tableIdArgument: true,
     },
+    UniqueEntity: {
+      directory: "/modules/uniqueentity/tables",
+      primaryKeys: {},
+      schema: "bytes32",
```

Comment:
now that this has a system that things would funnel through, I feel like it makes sense to move this back to `uint256` to avoid converting back/forth, then just have the system return `bytes32(incremented ID)`